### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/layout/navbar.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#607] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/layout/navbar.html

导航栏可能包含你的站点名称、搜索框、 导航栏链接、多语言支持、仓库链接 和 外观弹窗。它们均取决于你的配置。

https://theme-hope.vuejs.press/zh/guide/layout/navbar.html

**Comment by 1226085293** [comment]
下面是我的侧边栏配置，其中 `随笔` 分类的 `children: "structure"` 无效，prefix 路径正确，有什么问题？
```
		{
			text: "文章",
			prefix: "博客/",
			icon: "shuji1",
			// children: "structure",
			children: [
				{
					text: "随笔",
					prefix: "博客/随笔/",
					collapsible: true,
					children: "structure",
				},
			],
		},
```

**Reply by 1226085293** [reply]
已经解决了，prefix 应该为 "随笔/",

**Comment by zhuonixian** [comment]
可以自定义添加一个导航栏图标吗    内置组件之外的。

**Reply by Mister-Hope** [reply]
主题的导航栏不就在这样做么？

**Comment by la1den** [comment]
导航栏左侧该如何配置才能不显示`locales`中配置的`title`呢？

**Reply by la1den** [reply]
翻到了，把`navbarTitle`改了就行

**Comment by DUDU-dudu-DUDU** [comment]
如何设置网页标签卡的图标  这个真的搜了好一会

**Reply by Mister-Hope** [reply]
favicon
